### DISCUSSION TITLE
UseCase for Ajax Application recording with mockoon

### DISCUSSION THREAD

**Discussion by tWatanabeCTC** [mockoon/mockoon#1260] [Help]
we are reverse engineering now, i understand modify backend ajax request URL is very simple, however, before modifying url we need to make stab so we expect mockoon is very good tool for us.

When we use mockoon as simple API recording, it's work well because of it does not include need downloading js and css like browser.

In our case, I would like to record ajax API behavior, we need to user browser and have to download js to work ajax communication.

When we use mockoon proxy, Chrome looks like only making favicon download request that's why browser does not get JavaScript so they cannot render any page and they cannot have function to ajax requests.

We try to http://www.yahoo.co.jp/ (not my application, common web) it looks like work same behavior.

I guess our understanding is wrong or mockoon is not support for Ajax communition.
Do you have any solution or good document?

<img width="640" alt="image" src="https://github.com/mockoon/mockoon/assets/61773996/abfdb314-23bd-49b4-bc4c-4bbec329c398">

**Comment by 255kb** [comment]
I just tested to proxy a regular webserver with HTML/JS files through Mockoon and it worked.
My setup was `Chrome --> Mockoon localhost:3000 --proxy--> Http server localhost:5500`. Chrome downloaded both the HTML and the JS files. 
Do you get any error? Or do you have wildcard routes that could intercept all the calls?
![image](https://github.com/mockoon/mockoon/assets/7489814/9d7ffd6f-9b3c-4a32-a419-5cfd4309a62a)
